### PR TITLE
Nested namespace

### PR DESCRIPTION
**Pull request by DennisOSRM**
It's a mechanical change to modernize the code base

### CODE REVIEW

File: include/contractor/contract_excludable_graph.hpp [hunk 1]
```diff
@@ -94,7 +92,6 @@ inline auto contractExcludableGraph(ContractorGraph contractor_graph_,
     return GraphAndFilter{QueryGraph{num_nodes, edge_container.edges},
                           edge_container.MakeEdgeFilters()};
 }
-} // namespace contractor
 } // namespace osrm
```
Comment (SiarheiFedartsou) [inline include/contractor/contract_excludable_graph.hpp:95]: nit. would be great to reflect this in such comments too(if possible to do in automated fashion ofc)

**Review by SiarheiFedartsou — APPROVED** [review]

File: include/contractor/contract_excludable_graph.hpp (hunk 1 above)
Comment (DennisOSRM) [reply include/contractor/contract_excludable_graph.hpp:95]: fair point. I am afraid those are hard to do in an automated way, tho.

Comment (SiarheiFedartsou) [reply include/contractor/contract_excludable_graph.hpp:95]: Then I think it is not big problem if we keep it like this

Comment (DennisOSRM) [reply include/contractor/contract_excludable_graph.hpp]: I spoke too early. The latest push to this PR should fix the closing namespace annotation.

**Pull request merged**